v2_v3_outputs_map.csv

```diff
@@ -178,15 +176,16 @@ Financial,replacement_costs,Financial,replacements_future_cost_after_tax
 Financial,simple_payback_years,Financial,simple_payback_years
 Financial,year_one_om_costs_us_dollars,Financial,year_one_om_costs_after_tax
 Financial,year_one_om_costs_before_tax_us_dollars,Financial,year_one_om_costs_before_tax
----,(Added in v3),Financial,year_one_om_costs_before_tax_bau
 Financial,year_one_om_costs_before_tax_bau_us_dollars,Financial,year_one_om_costs_before_tax_bau
-Financial,lcoe_component_diesel_capex_us_dollars_per_kwh,Financial,
-Financial,lcoe_component_fuel_us_dollars_per_kwh,Financial,
-Financial,lcoe_component_om_us_dollars_per_kwh,Financial,
-Financial,lcoe_component_other_annual_costs_us_dollars_per_kwh,Financial,
-Financial,lcoe_component_other_capex_us_dollars_per_kwh,Financial,
-Financial,lcoe_component_re_capex_us_dollars_per_kwh,Financial,
 Financial,total_om_costs_bau_us_dollars,Financial (not yet in v3),lifecycle_om_costs_after_tax_bau
+---,(Added in v3),Financial (not yet in v3),year_one_om_costs_before_tax_bau

```

Comment:
@gellwood why is this repeated? (year_one_om_costs_before_tax_bau appears above as well)

Reply:
@adfarth I think that's a mistake.